### PR TITLE
AB#54896 set maximum pagination limit big requests can raise mongoDB …

### PR DESCRIPTION
**Pull request by ketanreliefapps**
# Description
in the config file, add a parameter for the max pagination
in all places that would use pagination, throw an error if page size is greater than 1000
error can be:
Maximum page size is 1000. Please use a different page size.
1000 should be injected from the config, so the text of the notification should be:
Maximum page size is {value}. Please use a different page size.
and you inject the value dynamically in the string

## Ticket
https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/54896

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (refactor or addition to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:
- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have commented my code, particularly in hard-to-understand areas
- [x] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/schema/query/pullJobs.query.ts [hunk 1]
```diff
@@ -18,6 +19,15 @@ export default {
     afterCursor: { type: GraphQLID },
   },
   async resolve(parent, args, context) {
+    const first = args.first || DEFAULT_FIRST;
```
Comment (AntoineRelief) [inline src/schema/query/pullJobs.query.ts:22]: @ketanreliefapps 
you can put that in a shared method, it's a lot of duplicated code

Comment (ketanreliefapps) [reply src/schema/query/pullJobs.query.ts:22]: Done

File: src/utils/schema/resolvers/Query/all.ts [hunk 2]
```diff
@@ -16,10 +16,29 @@ import { getAccessibleFields } from '@utils/form';
...
 /** Default number for items to get */
 const DEFAULT_FIRST = 25;
 
+/**
+ * Checks pagination maximum limit of data.
+ * Throw pagination maximum limit cross than error.
+ *
+ * @param maxLimit Question maxLimit number
+ */
+export const paginationMaxLimit = (maxLimit: number): void => {
```
Comment (AntoineRelief) [inline src/utils/schema/resolvers/Query/all.ts:31]: @ketanreliefapps 
this comment is just for information, you don't need to work on it
it does not make sense to put this code there
it is a shared method, not a specific code of the query/all
please make sure for next tickets to take that into account when deciding where to put code

Comment (ketanreliefapps) [reply src/utils/schema/resolvers/Query/all.ts]: Sorry, Okay

**Pull request merged**